front-end/src/UserHomePage.js

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+import TopNavigation from "@cloudscape-design/components/top-navigation";
+import Container from "@cloudscape-design/components/container";
+import Header from "@cloudscape-design/components/header";
+import ContentLayout from "@cloudscape-design/components/content-layout";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+import Button from "@cloudscape-design/components/button";
+import Badge from "@cloudscape-design/components/badge";
+
+export default function UserHomePage (){
+  const [data, setData] = React.useState([
+    { title:"Employee survey 1", due: ('2023-10-20'),id:1},
+    { title:"Employee survey 2", due: ('2023-09-30'),id:2},
+    { title:"Employee survey 3", due: ('2023-09-12'),id:3}
+  ]);
+
+  const today = new Date();
+  const month = today.getMonth()+1;
+  const year = today.getFullYear();
+  const date = today. getDate();
+  const currentDate = year + "/" + month + "/" + date;
+  console.log(currentDate);
+
+  return (
+    <React.Fragment>
+    <TopNavigation
```

Comment:
Isn't this top navigation common to all pages? It's better to create a new component for this and reuse the code for all the components. In App.js file instead of doing the current `<div className='...'><UserHomePage />...` we can do something like the following. In the following example `Header` is your new TopNavigation component that you will create. 

```
    <Header>
      <UserHomePage />
      ...
    </Header>
```

Reply:
I created a separate Header component but when I do this the app is not rendering UserHomePage.
```
    <Header>
      <UserHomePage />
      ...
    </Header>
```
So I did this,
``` 
return (
    <div>
      <Header />      
      <UserHomePage />
    </div>
  );
```
Also, I seem to get an error when I don't include the div tag - **JSX expressions must have one parent element**